src/util/Options.js

```diff
@@ -135,7 +135,6 @@ class Options extends null {
         version: 9,
       },
       http: {
-        agent: {},
```

Comment:
Why was this removed?

Reply:
Probably some conflicts I came across when doing this. Thought I undid them all bug I’ll add this back

Reply:
Amended the commit